static_src/test/unit/stores/service_instance_store.spec.js

```diff
@@ -286,16 +286,35 @@ describe('ServiceInstanceStore', function() {
     });
   });
 
-  describe('on instance create error', function() {
-    it('should set create error to passed in error', function() {
-      var expectedError = { status: 500 };
+  describe('on SERVICE_INSTANCE_CREATE_ERROR', () => {
+    it('should set `createError` based on error received', () => {
+      const serverError = { code: 500 };
+      const argumentError = { response: { data: { error_code: 'CF-MessageParseError' } } };
+      const spaceError = { response: { data: { error_code: 'CF-ServiceInstanceInvalid' } } };
+      const serverErrorMsg =
+        'Error #500: please contact cloud.gov support for help troubleshooting this issue.';
```

Comment:
let me know if you think this would be something for another PR:  would be nice to remove the `cloud.gov` hard coding so that in the future we can achieve https://github.com/18F/cg-dashboard/issues/790 eventually